packages/e/exprtk/pspec_x86_64.xml

```diff
@@ -24,12 +24,12 @@
         </Files>
     </Package>
     <History>
-        <Update release="1">
-            <Date>2023-12-09</Date>
-            <Version>2.71</Version>
-            <Comment>Packaging update</Comment>
+        <Update release="2">
+            <Date>2024-01-01</Date>
+            <Version>0.0.3</Version>
+            <Comment>Update to version 0.0.3</Comment>
             <Name>Alexander Vorobyev</Name>
```

Comment:
One more thing: This line should have changed. Did you set up a `packager` file? See here: https://help.getsol.us/docs/packaging/prepare-for-packaging/#setting-up-the-packager-file